src/ui/MobileFacet/MobileFacet.ts

```diff
@@ -0,0 +1,57 @@
+import { Component } from '../Base/Component';
+import { ComponentOptions } from '../Base/ComponentOptions';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { exportGlobally } from '../../GlobalExports';
+import { Utils } from '../../Core';
+
+export interface IMobileFacetOptions {
+  breakpoint?: number;
+  isModal?: boolean;
+  showBackgroundWhileOpen?: boolean;
+  lockScroll?: boolean;
+}
+
+/**
+ * Allows the customization of how facets are displayed to mobile users.
+ */
+export class MobileFacet extends Component {
+  static ID = 'MobileFacet';
+
+  /**
+   * @componentOptions
+   */
+  static options: IMobileFacetOptions = {
+    /**
+     * Under which screen width facets should be in a mobile state and collapsed into a button.
+     */
+    breakpoint: ComponentOptions.buildNumberOption(),
```

Comment:
What would be the default value for this?

Reply:
By default, it inherits the same breakpoint as other responsive components.